Plugwise backend bump breaks the HA vilfo integration

Please see https://github.com/home-assistant/core/pull/113245

This is the error we're seeing during the testing of the above PR:
```
==================================== ERRORS ====================================
_________ ERROR collecting tests/components/vilfo/test_config_flow.py __________
ImportError while importing test module '/home/runner/work/core/core/tests/components/vilfo/test_config_flow.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/opt/hostedtoolcache/Python/3.11.8/x64/lib/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/components/vilfo/test_config_flow.py:4: in <module>
    import vilfo
venv/lib/python3.11/site-packages/vilfo/__init__.py:5: in <module>
    from .client import (
venv/lib/python3.11/site-packages/vilfo/client.py:7: in <module>
    from semver import compare as semver_compare
E   ModuleNotFoundError: No module named 'semver'
```
It is caused by the plugwise-backend module removing the `semver` dependency in this PR, which breaks the vilfo-backend module.

Please also see frenck's comment in the Core PR.

--- a/setup.cfg
+++ b/setup.cfg
@@ -16,6 +16,7 @@ packages = find:
 install_requires =
     requests
     getmac
+    semver
 setup_requires =
     setuptools_scm
 

